README.md

```diff
@@ -1,161 +1,720 @@
+
+    %% Transative Path Styles
+        %% baklava -> chocolate -> avodcado
+           linkStyle 13 stroke: DodgerBlue;
+           linkStyle 28 stroke: DodgerBlue;
+
+        %% calamari -> bean -> adobo
+            linkStyle 11 stroke: orange;
+            linkStyle 14 stroke: orange;
+            linkStyle 16 stroke: orange;
+
+        %% ambrosia -> cinnamon -> cake -> agave
+            linkStyle 8  stroke: deeppink;
+            linkStyle 24 stroke: deeppink;
+            linkStyle 25 stroke: deeppink;
+```
+
+Note the direction of the arrows: they point from an event to their antecedent cause (i.e. they are the directed `cause` pointers in EVAC quads). This is sometimes counterintuitive when first encountered, since we normally reason from cause to effect. However, all facts in PomoDB are by their nature "in the causal past". Items in a node's causal history are called its "ancestors". Nodes that depend on a fact are called its "descendants". Graphs MAY be rooted.
+
+#### 4.2.2.1 Genesis Nodes
+
+The "earliest" facts in the graph above are `bafy...almond` and `bafy...bacon` (highlighted in blue), as they have no asserted causes. These are called "genesis nodes" (or simply "geneses"). In formal terms, these are "causal sinks". These nodes MAY be treated as objectively the earliest in the graph: since as causal information is asserted at write-time, these cannot be earlier, unlisted facts added or discovered later.
+
+There MUST be at least one genesis node in an inhabited graph (even if it is a single-node graph). There MAY be an unbounded number of concurrent genesis nodes.
+
+#### 4.2.2.2 Head Nodes
+
+The "latest" facts in the above graph are `bafy...berry`, and `bafy...coffee` (highlighted in green). These are called "head nodes", as they have no known descendants at read-time. In formal terms, these are "causal sources". These nodes are merely subjectively the most recent fact: it is possible that some other facts were written elsewhere, but simply have not arrived at the reader yet. Since more facts MAY be appended to the history at any time.
+
+There MUST be at least one head node in an inhabited graph (even if it is a single-node graph). There MAY be an unbounded number of concurrent head nodes.
+
+#### 4.2.2.3 Concurrent Nodes
+
+When comparing any two nodes, if one does not appear in the causal history of the other, then they are said to be "concurrent" or "sibling" nodes. As any two causally disjoint graphs have (by definition) no dependencies on each other, most facts are concurrent.
+
+#### 4.2.2.4 Parent Nodes
+
+Nodes listed in the [`Cause`] field of a fact are said to be that fact's "parent nodes". A fact MUST have zero or more parents.
+
+``` mermaid
+flowchart BT
+    parentA
+    parentB
+    
+    fact --> parentA
+    fact --> parentB
+```
+
+#### 4.2.2.5 Child Nodes
+
+Nodes that list a fact is their [`Cause`] field of a fact are said to be that fact's "child nodes". A fact MUST have zero or more children.
```

Comment:
s/Nodes that list a fact is their/Nodes that list a fact in their

Reply:
s/are said to be that/are said to be among that

Reply:
Thanks for the corrections! Have you seen the `suggestion` feature in GitHub? It may be easier than `sed` syntax

Reply:
> s/are said to be that/are said to be among that

I'm less certain about this one. The ones listed are the exact set of parents (and no others), or am I misunderstanding?